.github/workflows/test.yml

```diff
@@ -84,18 +97,27 @@ jobs:
           # Pushes tag - deploy tag name.
           echo "TAG=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
         fi
-        echo "DOCKER_BASE=${DOCKER_USER}/${DOCKER_REPO}" >> $GITHUB_ENV
-    - name: Install Docker buildx
+        echo "DOCKER_BASE=${{ secrets.DOCKER_USER }}/${{ env.DOCKER_REPO }}" >> $GITHUB_ENV
+    - name: Build and push
+      id: buildx
```

Comment:
why do we need this id?

Reply:
Not used, was part of a previous test, good catch!